source/lac/scalapack.cc

```diff
@@ -407,8 +407,18 @@ ScaLAPACKMatrix<NumberType>::copy_to (ScaLAPACKMatrix<NumberType> &dest) const
       ierr = MPI_Group_union(group_source, group_dest, &group_union);
       AssertThrowMPI(ierr);
       MPI_Comm mpi_communicator_union;
-      // to create a communicator representing the union of the source and destination MPI communicator
-      // we need a communicator containing all  desired processes --> use MPI_COMM_WORLD
+      
+      // to create a communicator representing the union of the source
+      // and destination MPI communicator we need a communicator that
+      // is guaranteed to contain all desired processes -- i.e.,
+      // MPI_COMM_WORLD. on the other hand, as documented in the MPI
+      // standard, MPI_Comm_create_group is not collective on all
+      // processes in the first argument, but instead is collective on
+      // only those processes listed in the group. in other words,
```

Comment:
In 

(capital)

Reply:
ah, we do this everywhere in the documentation that we start sentences with lower-case letters...

Reply:
hm, never noticed this, nevermind then

Reply:
... and I have been trying to fight this for years but @bangerth keeps adding more and more of these. ;-)

Reply:
Alright alright, I'll change my ways in the future.